code/_onclick/hud/robot.dm

```diff
@@ -342,3 +342,43 @@
 	if(.)
 		return
 	robot.modularInterface?.interact(robot)
+
+//borg pda
+/datum/hud/robot/New(mob/owner)
```

Comment:
I should definitely ask why this wasn't add to the New above instead of making another one